.github/ISSUE_TEMPLATE/error-report.md

```diff
@@ -0,0 +1,18 @@
+---
+name: "\U0001F41B Error Report"
+about: Typos, grammatical errors, terminology misuse, ambiguity, etc.
+---
+
+## Error Report
+
+**This repository is ONLY used to solve issues related to DOCS.
+For other issues (related to TiDB, TiKV, etc), please move to [other repositories](https://github.com/pingcap/).**
+
+Please answer the following questions before submitting your issue. Thanks!
+
+1. What is the directory of the document related to this issue?
```

Comment:
Do we want to know the directory, or a url/path?

Reply:
Yes, the URL/path of the doc is what we want more. I'll change it.